### ISSUE TITLE
 Ghidra causes gnome-shell to use 100% CPU

### ISSUE THREAD

**Issue by Dylan-Brotherston** [NationalSecurityAgency/ghidra#780]
**Describe the bug**
Running ghidra causes gnome-shell to use 100% CPU

**To Reproduce**
Steps to reproduce the behaviour:
1. Launch ghidra with "./ghidraRun"
2. All GUIs become unresponsive.
3. sending 2 SIGQUITs to gnome-shell allows recovery without a restart.

Stderr from './launch.sh debug Ghidra 768M "--illegal-access=warn" ghidra.GhidraRun':
```
OpenJDK 64-Bit Server VM warning: Archived non-system classes are disabled because the java.system.class.loader property is specified (value = "ghidra.GhidraClassLoader"). To use archived non-system classes, this property must not be set
openjdk version "12.0.1" 2019-04-16
OpenJDK Runtime Environment (build 12.0.1+12-Ubuntu-1)
OpenJDK 64-Bit Server VM (build 12.0.1+12-Ubuntu-1, mixed mode, sharing)
WARNING: Illegal reflective access by ghidra.framework.GhidraApplicationConfiguration (file:/home/dylan/Downloads/ghidra_9.0.4/Ghidra/Features/Base/lib/Base.jar) to field sun.awt.X11.XToolkit.awtAppClassName
```

**Environment**
 - OS: Ubuntu 19.04
 - Java Version: openJDK 12 (default, openJDK 11 installed)
 - Ghidra Version: 9.0.4

**Comment by hedgeberg** [comment]
Cannot reproduce on an up-to-date Arch machine. It's possible this is an issue in gnome-shell that has already been resolved upstream and the update just hasn't made it to ubuntu yet? Do you know of any other ubuntu users dealing with the same issue?

**Comment by Dylan-Brotherston** [comment]
I am running
```shell
$ gnome-shell --version
GNOME Shell 3.32.2
```

**Comment by ryanmkurtz** [comment]
I've run under Ubuntu 19.04 in both a VirtualBox VM and bare metal and haven't noticed this issue.  Have you changed any of the default properties in `support/launch.properties`?

**Comment by Dylan-Brotherston** [comment]
I am using a fresh install.
After checking other programs it looks like this is either a gnome-shell or a java problem.
This issue is also affecting other java, and only java, programs that were working before installing java 11.
Probably best to close this issue. I am going to open an issue with gnome-shell.

**Comment by ryanmkurtz** [comment]
Ok, thanks for following up.

**Comment by ryanmkurtz** [comment]
Possibly related to #653

**Comment by Dylan-Brotherston** [comment]
Thank you for the link.
This very interesting as JetBrains IDEs are the other programs not working for me.
Unfortunately his solution won't work as GNOME Shell 3.32.1 is my current version.

**Comment by Dylan-Brotherston** [comment]
Just wanted to give an update.
This is a known issue for other java based programs.
It is caused by and tracked by the mutter WM [here](https://bugs.launchpad.net/ubuntu/+source/mutter/+bug/1834583).
There is a fix for Ubuntu 19.10(eoan).
For Ubuntu 19.04(dingo) use the workaround [here](https://youtrack.jetbrains.com/issue/JBR-1644)
